Funcion Main

zaballin opened the pull request:
Nuevo main con las correccion de encabezados de suma y resta

zaballin committed:
OLA K ASE se agrega la funcion main gracias

zaballin committed:
Main con correcciones

blancogerardo94 commented on main.c:
```diff
@@ -1,37 +1,112 @@
+                printf("dame un valor\n");
+                scanf("%f",&f1);
+                res = coseno(f1);
+                printf("EL resultado es: %d\n",res);
```
%d imprime un decimal, para este caso se necesita %f

blancogerardo94 commented on main.c:
```diff
@@ -1,37 +1,112 @@
+                scanf("%d",&val1);
+                scanf("%d",&val2);
+                res = potencia(val1,val2);
+                printf("La resultado es %d\n",res);
```
%d imprime un decimal, para este caso se necesita %f

Luisvizk commented on main.c:
```diff
@@ -1,37 +1,112 @@
+     {
+        printf("Menu\n");
+        printf("Opcion \n");
+        printf("1-Suma \n2-Resta \n3-Multiplica\n4-Pi\n5-Exponencial\n6-Potenca\n7-Coseno\n8-Division\n9-Serie de Taylor\n");
```
Serie de Taylor comprende muchas acciones matemáticas distintas, mejor llamarla seno y ya.

Luisvizk commented on main.c:
```diff
@@ -1,37 +1,112 @@
+                printf("dame dos valores\n");
+                scanf("%f",&f1);
+                scanf("%d",&val2);
+                res = serietaylor(f1,val2);
```
la función dentro del modulo se llama "sen_tay"

Luisvizk commented on main.c:
```diff
@@ -1,37 +1,112 @@
+                break;
+           default:
+                  printf ("opcion no valida \n");
+          }
```
Una recomendación, para mejorar el código regresar los resultados a una variable, pero en ves de imprimirla inmediatamente, imprimirla después de los casos, para ahorrar espacio, algo como "El resultado de la operación es: "

zaballin replied:
"res = serietaylor(f1,val2);
-                printf("El resultado es %d\n",res); <------- Pues así no?
-                break;

KariAvila commented on main.c:
```diff
@@ -1,37 +1,112 @@
+                res = expo(f1);
+                printf("EL resultado es: %d\n",res);
+                break;
+	case 6:
```
Perfecto ;)

zaballin committed:
Update main.c

zaballin commented:
@camaleon6994 @charis38 @Luisvizk @KariAvila @feri2603 Chavos, vi sus códigos y corrí los que puede y a los demás les comente el código de su ultimo _commit_, si ya sirven sus funciones revisen que en este ultimo commit que hice de main sus encabezados estén correctos (según yo si están) cualquier cosa coméntenme y los corrijo para ya pedirle a @eortizl que nos acepte los requests. Saludos a todos! ÁNIMO!!

zaballin committed:
Update main.c

zaballin commented on main.c:
```diff
@@ -62,7 +62,7 @@ int main(void)
                 scanf("%d",&val2);
                 res_potencia = potencia(val1,val2);
-                printf("La resultado es %d\n",res);
+                printf("La resultado es %lu\n",res_potencia);
```
@KariAvila lo corrí en xcode y sin broncas, tu a la hora de imprimri para porbar q funcionrá usaste %lu??

charis38 commented on main.c:
```diff
@@ -30,58 +31,58 @@ int main(void)
 	case 4:
                 res = calculaPi();
-                printf("PI es igual a: %d\n",res);
+                printf("PI es igual a: %f\n",res);
```
Por lo que veo esto funciona bien, falta hacer el merge y comprobar que funcione el programa con todo junto.

eortizl commented on main.c:
```diff
@@ -1,37 +1,113 @@
 extern float suma(int a, int b);
-extern float mult(int a, int b);
+extern float resta(int a, int b);
+/** 
```
Estos comentarios no deben ir, sólo debe incluirse lo que ya existe

zaballin replied:
Pero si están en mi repositorio, aunque el comitt lo haga sólo del mail también van el resto de archivos no? Igual lo cortijo

eortizl commented on main.c:
```diff
@@ -1,37 +1,113 @@
+extern float serietaylor(float a, int b);
+ **/
 
+int menu(void);
```
Deberia ir junto a los otros externs, separado por lineas al menos dos, antes de main

eortizl commented on main.c:
```diff
@@ -1,37 +1,113 @@
-	}
-	else
+}
+int menu(void)
```
menu debe estar en otro archivo, no esta bien identado

eortizl commented on main.c:
```diff
@@ -1,37 +1,113 @@
+          switch (opc)
+          {
+        case 1:
+                printf("dame dos valores\n");
```
Debe ser mas especificos, usa entonces dos printf - dame un valor entero, scanf, luego otro printf, dame el segundo valor entero, scanf. Seria bueno que hubiera mensajes de que la opcion es para hacer una suma

eortizl commented on main.c:
```diff
@@ -1,37 +1,113 @@
+                printf("La suma es %f\n",res);
+                break;
+	case 2:
+                printf("dame dos valores\n");
```
igual que en el comentario de arriba, 2 printf, 2 scanf, di la opcion

eortizl commented on main.c:
```diff
@@ -1,37 +1,113 @@
+                printf("La resta es %f\n",res);
+                break;
+        case 3:
+                printf("dame dos valores\n");
```
Igual que en el comentario anterior

eortizl commented on main.c:
```diff
@@ -1,37 +1,113 @@
+           default:
+                  printf ("opcion no valida \n");
+          }
+        }while(opc!=9);
```
No es la opcion de salida, en todo caso debes hacer la salida con la opcion de salida, en el switch debes ver que este bien. Si doy opcion 11 me va a decir que no es valida y se va a salir

zaballin committed:
Update main.c

zaballin committed:
Función menu en otro archivo.

Todas las funciones estan como extern, el menu va en un nuevo archivo, todas las opciones especifican que es y pide los valore spor separado si es el caso @eortizl

zaballin committed:
Menu separado

Seperamos la funcion menu del main

zaballin committed:
Update main.c

zaballin committed:
IDENTADO PAPALOY

zaballin committed:
Update main.c

zaballin commented on main.c:
```diff
@@ -2,110 +2,116 @@
 #include <math.h>
```
@KariAvila A ver ocmo lo vez???
